grails-forge-core/src/main/java/org/grails/forge/feature/github/workflows/templates/javaSetup.rocker.raw

```diff
@@ -5,9 +5,9 @@
 JdkVersion jdkVersion
 )
 
-      - uses: actions/checkout@@v2
+      - uses: actions/checkout@@v4
       - name: Set up JDK @jdkVersion.majorVersion()
-        uses: actions/setup-java@@v2
+        uses: actions/setup-java@@v4
         with:
           distribution: 'adopt'
```

Comment:
The distribution should now be 'temurin' according to https://github.com/actions/setup-java:
>NOTE: AdoptOpenJDK got moved to Eclipse Temurin and won't be updated anymore. It is highly recommended to migrate workflows from adopt and adopt-openj9, to temurin and semeru respectively, to keep receiving software and security updates. See more details in the [Good-bye AdoptOpenJDK post](https://blog.adoptopenjdk.net/2021/08/goodbye-adoptopenjdk-hello-adoptium/).

We should take this opportunity to update it.